vscode/webviews/Chat.tsx

```diff
@@ -269,6 +273,7 @@ const TextArea: React.FunctionComponent<ChatUITextAreaProps> = ({
 }) => {
     const inputRef = useRef<HTMLTextAreaElement>(null)
     const placeholder = 'Message (@ to include code, / for commands)'
+    const disabledPlaceHolder = 'Chat and commands have been disabled by your Enterprise instance site administrator'
```

Comment:
```suggestion
    const disabledPlaceHolder = 'Chat and commands have been disabled by your Sourcegraph instance site admin'
```

I think this text should change depending on what is disable, right?

* Chat AND commands are disabled -> `Chat and commands have been disabled by your Sourcegraph site admin`
* Only Chat is disabled -> `Chat has been disabled by your Sourcegraph site admin`

Reply:
Also, an edge case I had not thought about: if chat is enabled, but commands are disabled.. how does the user know commands are disabled? where do we tell them?

Reply:
Currently we say this 
https://sourcegraph.com/github.com/sourcegraph/cody@ara/site-config/-/blob/vscode/src/chat/chat-view/ChatManager.ts?L97%3A18-97%3A34 

But instead of saying this feature I can also say commands disabled. 
![image](https://github.com/sourcegraph/cody/assets/11155207/95a00aaa-c057-4b87-9f71-aa1f7461a893)
The UI looks like this. 

Chat disabled commentary and command disabled commentary are separate things so they should be expressed separately in the UX. That was my thinking

Reply:
I fixed the wording to just say "Chat has been disabled by your Sourcegraph instance site admin "